zmon-controller-ui/views/trialRun.html

```diff
@@ -2,7 +2,7 @@
     <div class="row">
         <div class="col-md-8">
             <h1 class="no-margin">Trial Run</h1>
-            <p><a href="/docs/check-definitions.html#trial-run">Trial Run</a> allows the end user to test check commands and alert conditions before using them on the system.</p>
+            <p><a href="https://docs.zmon.io/">Trial Run</a> allows the end user to test check commands and alert conditions before using them on the system.</p>
```

Comment:
https://docs.zmon.io/en/latest/user/check-definitions.html#using-trial-run

Not documented yet.